cdap-ui/app/services/adapter/myadapterapi.js

```diff
@@ -1,11 +1,12 @@
-angular.module(PKG.name + '.feature.adapters')
-  .factory('AdapterApiFactory', function($resource, myHelpers, myCdapUrl) {
+angular.module(PKG.name + '.services')
+  .factory('myAdapterApi', function($resource, myHelpers, myCdapUrl) {
     var url = myCdapUrl.constructUrl,
         templatePath = '/templates',
-        adapterPath = '/namespaces/:namespace/adapters/:adapter'
+        adapterPath = '/namespaces/:namespace/adapters/:adapter',
```

Comment:
we have ```save``` as ```/adapters/:adapterId``` shouldn't our get too be ```/adapter/:adapterId```

When I saw ```myAdapterApi.get(params)``` I thought we were getting the details of an adapter. Should we move this out to a separate resource file? Because we are having save, get, delete, list and only save uses a different url (however we can GET details of the adapter using the url that we use for save too).